common/config/src/main/java/io/helidon/common/config/Config.java

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright (c) 2018, 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.config;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * <h2>Configuration</h2>
```

Comment:
Adding headers to javadocs is tricky and not really needed here, when we have a single sentence description